source/ShopPanel.h

```diff
@@ -117,10 +120,13 @@ class ShopPanel : public Panel {
 	const Outfit *selectedOutfit = nullptr;
 	
 	double mainScroll = 0.;
-	double sideScroll = 0.;
+	double sideShipScroll = 0.;
+	double sideDetailScroll = 0.;
 	double maxMainScroll = 0.;
-	double maxSideScroll = 0.;
-	bool dragMain = true;
+	double maxSideShipScroll = 0.;
+	double maxSideDetailScroll = 0.;
+	bool hoverInfo = false;
```

Comment:
Rather than use two booleans to indicate which panel is being hovered / manipulated, let's define a class enum with 3 values, e.g.
```c++
enum class ShopPane : int {
  Main,
  Sidebar,
  Info
};
```
and then we need only one variable of that type, e.g. `activePane`, whose value will be e.g. `ShopPane::Main`.

Reply:
Thank you, refactored in 376eac29.